bench-tps/src/main.rs

```diff
@@ -83,10 +83,16 @@ fn create_connection_cache(
     client_node_id: Option<&Keypair>,
 ) -> ConnectionCache {
     if !use_quic {
-        return ConnectionCache::with_udp(tpu_connection_pool_size);
+        return ConnectionCache::with_udp(
+            "connection_cache_bench_tps_udp",
```

Comment:
bench-tps metrics are usually named like `bench-tps-.*`, so it is simpler to search based on the bench client name.
Outside of scope of this PR, would be generally good to agree on some convention about metrics names (snake case vs kebab, start with capital letter or not etc)

Reply:
renamed to `bench-tps-*`